packages/framework/src/Model/Product/Search/ProductElasticsearchConverter.php

```diff
@@ -60,6 +60,11 @@ public function fillEmptyFields(array $product): array
         $result['name_prefix'] = $product['name_prefix'] ?? null;
         $result['name_suffix'] = $product['name_suffix'] ?? $product['name_sufix'] ?? null;
 
+        $result['availability_status'] = $product['availability_status'] ?? '';
+        $result['store_availabilities_information'] = $product['store_availabilities_information'] ?? [];
+        $result['available_stores_count'] = $product['available_stores_count'] ?? null;
+        $result['stock_quantity'] = $product['stock_quantity'] ?? null;
```

Comment:
just a thought for future improvement - this method may also use the constants from the productExportFieldProvider, or?